Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/validation/ui/validationresult/common/listitem/ValidationInputVH.kt

```diff
@@ -0,0 +1,51 @@
+package de.rki.coronawarnapp.covidcertificate.validation.ui.validationresult.common.listitem
+
+import android.view.ViewGroup
+import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.covidcertificate.validation.core.DccValidation
+import de.rki.coronawarnapp.databinding.CovidCertificateValidationResultInputItemBinding
+import de.rki.coronawarnapp.util.TimeAndDateExtensions.toShortDayFormat
+import de.rki.coronawarnapp.util.TimeAndDateExtensions.toShortTimeFormat
+import de.rki.coronawarnapp.util.TimeAndDateExtensions.toUserTimeZone
+import de.rki.coronawarnapp.util.lists.diffutil.HasPayloadDiffer
+
+class ValidationInputVH(
+    parent: ViewGroup
+) : BaseValidationResultVH<ValidationInputVH.Item, CovidCertificateValidationResultInputItemBinding>(
+    R.layout.covid_certificate_validation_result_input_item,
+    parent
+) {
+
+    override val viewBinding = lazy {
+        CovidCertificateValidationResultInputItemBinding.bind(itemView)
+    }
+
+    override val onBindData: CovidCertificateValidationResultInputItemBinding.(
+        item: Item,
+        payloads: List<Any>,
+    ) -> Unit = { item, payloads ->
+
+        val arrivalDateString = item.validation.userInput.arrivalAt.toUserTimeZone().run {
+            "${toShortDayFormat()}, ${toShortTimeFormat()}"
```

Comment:
⏬ there is no comma between date and time in figma imho
please check

Reply:
comma removed